tests/tests.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+from unittest import TestCase
+import json
+
+from openfisca_web_api.app import create_app
+
+tester = create_app('openfisca_dummy_country').test_client()
```

Comment:
I like to call my [SUT](https://en.wikipedia.org/wiki/System_under_test) “subject”.